js/main.js

```diff
@@ -1,8 +1,70 @@
+const PICTURE_COUNT = 25;
+const MAX_COMMENTS = 10;
+const MIN_LIKES = 15;
+const MAX_LIKES = 200;
+const AVATAR_COUNT = 6;
+const NAMES = ['Никита', 'Андрей', 'Константин', 'Павел', 'Алина', 'Ксюша'];
+const SENTENSES = [
+  'Всё отлично!',
+  'В целом всё неплохо. Но не всё.',
+  'Когда вы делаете фотографию, хорошо бы убирать палец из кадра. В конце концов это просто непрофессионально.',
+  'Моя бабушка случайно чихнула с фотоаппаратом в руках и у неё получилась фотография лучше.',
+  'Я поскользнулся на банановой кожуре и уронил фотоаппарат на кота и у меня получилась фотография лучше.',
+  'Лица у людей на фотке перекошены, как будто их избивают. Как можно было поймать такой неудачный момент?!'
+];
+
 const random = (min, max) => {
   return Math.floor(Math.random() * (max - min + 1)) + min;
 };
 
-
 const checkLength = (str, maxLength) => {
   return str.length < maxLength;
 };
+
+const generatePictures = (count) => {
+  const pictures = [];
+    for (let i = 0; i < count; i++) {
```

Comment:
Слишком много пробелов
```suggestion
  for (let i = 0; i < count; i++) {
```